common/djangoapps/student/tests/test_refunds.py

```diff
@@ -65,13 +65,9 @@ def test_refundable(self):
         self.assertTrue(self.enrollment.refundable())
 
     def test_refundable_expired_verification(self):
-        """ Assert that enrollment is not refundable if course mode has expired."""
+        """ Assert that enrollment is refundable if course mode has expired."""
```

Comment:
i think this is not the expected behaviour because the fix, this test is ensuring that user will not be refunded if the course mode is expired. now, after taking the product opinion, the expected behaviour is to refund the user whether the course mode is expired or refund_cutoff_date is greater than the course end date.